README.md

```diff
@@ -1,57 +1,202 @@
 
-- OS
-- Library version
-- ...
+> [!IMPORTANT]
+> This sample creates several resources. Delete the resource group after testing to minimize charges.
 
-### Installation
 
-(ideally very short)
+## Run in your local environment
 
-- npm install [package name]
-- mvn install
-- ...
+The project is designed to run on your local computer, provided you have met the  [required prerequisites](#prerequisites). You can run the project locally in these environments:
 
-### Quickstart
-(Add steps to get up and running quickly)
++ [Using Visual Studio Code](#using-visual-studio-code)
++ [Using Azure Functions Core Tools (CLI)](#using-azure-functions-core-tools-cli)
 
-1. git clone [repository clone url]
-2. cd [repository name]
-3. ...
+### Prerequisites
 
++ [Python 3.11](https://www.python.org/downloads/) 
++ [Azure Functions Core Tools](https://learn.microsoft.com/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools)
++ Install [Docker](https://www.docker.com/)
++ Install [Azurite storage emulator](https://learn.microsoft.com/azure/storage/common/storage-use-azurite). 
++ Clone the repo
 
-## Demo
+### Deploy language models
 
-A demo app is included to show how to use the project.
+1. [Create an Azure subscription with a valid payment method](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go). Free or trial Azure subscriptions *won't* work. 
 
-To run the demo, follow these steps:
+1. [Create a project in Azure AI Foundry](https://learn.microsoft.com/azure/ai-studio/how-to/create-projects?tabs=ai-studio)
 
-(Add steps to start up the demo)
+1. Go to **Model catalog** on the left menu and search for the following models to deploy:
+    - GPT-4
+    - GPT-3.5-turbo
+    - GPT-4o-mini
+    - Phi-4 ([small language model by Microsoft](https://techcommunity.microsoft.com/blog/aiplatformblog/introducing-phi-4-microsoft%E2%80%99s-newest-small-language-model-specializing-in-comple/4357090) that has advanced reasoning capabilities in areas like math and science)
+
+    ![Screenshot of model catelog on Azure AI Foundry](./media/model-catalog.png)
+  
+
+### Get endpoints and keys for models
+
+You'll need the model API key and endpoint for the next step.
+
+Go to the **Overview** tab of the project where models are deployed. API key is on the top.
+
+To get the endpoint, click on **Azure AI inference** under "Included capabilities":
+
+  ![Connection info](media/connection-info.png)
+
+### Set up Durable Task Scheduler emulator 
+
+1. Pull Docker image:
+    ```bash
+    docker pull mcr.microsoft.com/dts/dts-emulator:v0.0.5
```

Comment:
Should the instructions use `latest` instead of `v0.0.5` so that any fixes we make in the future will still be applied?

Reply:
actually turns out we haven't designated a `latest` tag yet. Phil just created an item tracking this.